tests/CMakeLists.txt

```diff
@@ -1,15 +1,10 @@
 project(hipsycl-tests)
 cmake_minimum_required(VERSION 3.5)
```

Comment:
These changes you should probably remove from this PR to avoid merge conflicts with your other PR